x-pack/docs/en/rest-api/security/get-role-mappings.asciidoc

```diff
@@ -23,7 +23,12 @@ Retrieves role mappings.
 ==== {api-description-title}
 
 Role mappings define which roles are assigned to each user. For more information, 
-see <<mapping-roles>>. 
+see <<mapping-roles>>.
+
+The role mapping APIs are generally the preferred way to manage role mappings
```

Comment:
```suggestion
The <<security-api-put-role-mapping>> is the preferred way to manage role mappings
```

Reply:
This talks about all the related APIs ( get/put/delete) instead of simply the put one